transcom/mymove
# B-17663 review expense parsing amount

## B-17663

## Summary

Updated ReviewExpense component to use shared util function `convertDollarsToCents` to properly parse dollar/cents amount, and updated the associated test.jsx file, to fix a bug where office users were unable to accept receipts whose dollar amounts exceeded $999.99

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. As a service counselor, view any customer move with a PPM shipment that's at the document review stage which includes a receipt.
2. Navigate to one of those receipts, and in the "Amount" field, enter an value equal or greater to $1,000 (so that a comma separator displays to the user).
3. Choose to accept/reject/exclude and press the "continue" button to progress to the next document. No errors should appear and you should navigate to the next document.
